tests/init.go

```diff
@@ -0,0 +1,60 @@
+package tests
+
+import (
+	"flag"
+	"regexp"
+	"strings"
+
+	"github.com/ipfs/gateway-conformance/tooling/specs"
+)
+
+type specsFlag string
+
+func (s *specsFlag) String() string {
+	return string(*s)
+}
+
+func (s *specsFlag) Set(value string) error {
+	names := strings.Split(value, ",")
+	var only, enable, disable = []specs.Spec{}, []specs.Spec{}, []specs.Spec{}
+	for _, name := range names {
+		spec, err := specs.FromString(regexp.MustCompile(`^[-+]`).ReplaceAllString(name, ""))
+		if err != nil {
+			return err
+		}
+		if strings.HasPrefix(name, "+") {
+			enable = append(enable, spec)
+		} else if strings.HasPrefix(name, "-") {
+			disable = append(disable, spec)
+		} else {
```

Comment:
If a spec is not prefixed with `+` nor `-`, we assume that the user wants to execute only that spec. Using flag `-specs=subdomain-gateway` would disable ALL the specs and then enable `subdomain-gateway` only.